### ISSUE TITLE
Vue: Export components from index.esm.js

### ISSUE THREAD

**Issue by emccorson** [OnsenUI/OnsenUI#2839]
In the ES module version of the Vue bindings, the index file does not export the components.

This is because we transpile the code to ES5, including the ES6 import statements, which would break tree shaking (i.e. all components would be included in the user's bundle, whether the user imported them or not).

After we start distributing in ES6 as per #2838, we should export the components from `index.esm.js` to let us use a less verbose syntax:

```
import { VOnsPage, VOnsToolbar, VOnsButton } from 'vue-onsenui/esm'
```

**IMPORTANT!** This must only be done after #2838.

**Comment by emccorson** [comment]
Closing this as something we will not do at the moment. It may be nice to have in the future.
